src/sentry/tasks/app_store_connect.py

```diff
@@ -54,8 +54,9 @@ def inner_dsym_download(project_id: int, config_id: str) -> None:
         return
 
     for i, (build, build_state) in enumerate(builds):
-        with sdk.configure_scope() as scope:
-            scope.set_context("dsym_downloads", {"total": len(builds), "completed": i})
+        sdk.Scope.get_isolation_scope().set_context(
+            "dsym_downloads", {"total": len(builds), "completed": i}
+        )
```

Comment:
@sentrivana, I was thinking we might be able to reuse the `scope` local variable from above here; however, to maintain the same functionality as the original, I replaced the `configure_scope` here with another call to `get_isolation_scope`.

Do you see any reason why the isolation scope could change from when the `scope` local variable was set? If not, should we reuse that variable here, or keep the code the way I have it now?

Reply:
I _think_ the isolation scope should not change here, but we can keep it as you have it now to be on the safe side.